app/javascript/articles/Article.jsx

```diff
@@ -133,6 +134,7 @@ export const Article = ({
                   article={article}
                   isBookmarked={isBookmarked}
                   onClick={bookmarkClick}
+                  saveable={saveable}
```

Comment:
I was going to suggest we skip rendering this button altogether if it's not saveable, but then I noticed for _some reason_ we also use `SaveButton` to render a follow button 🤯   I'm gonna put that on my list of "strange things in our JS we want to change at some point" 😄